data/sources/21_pois.yaml

```diff
@@ -0,0 +1,28 @@
+parabelrutsche:
+  parent: "mi"
+  name: "Parabelrutsche"
+  coords: { lat: 48.26254, lon: 11.66796 }
+
+  usage:
+    name: "Gravitatives Transportmittel"
+
+  props:
+    generic:
+      - name:
+          de: "Formel"
+          en: "Formula"
+        text: "z = y = hx²/d²"
+
+# === Validierungsautomaten ===
+validierungsautomat-1:
+  parent: "5101"
+  name: "Validierungsautomat 1 (Physik)"
+  coords: { lat: 48.267097, lon: 11.675607 }
+
+  usage:
+    name: "Validierungsautomat"
```

Comment:
How should these things be internationalized?
I think we could run `usage` through our `_(...)` translation function?